packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
We can't use callback in this case.
> The primary purpose for callbacks is to allow you to capture data from the test while its running, but they also provide an additional opportunity to add commands to the command queue

The benefits of callback: 
1. access to the result data.
2. add commands to the command queue.

We can not access the `browser.url('....')` object. We can only use `browser` object and `result` data object. And it adds command to commands queue, like:
```javascript
    browser
      .url(`http://url`)
      .waitForElementVisible('select', 1000)
      .click('select option[value="es"]')
      .assert.containsText('div', esTranslations['Terra.ajax.error']);
```
We still can't tell if all required data is loaded.

An alternative approach is using custom commands assertions, like `waitForText`, in [here](https://github.com/maxgalbu/nightwatch-custom-commands-assertions/blob/master/docs/waitForText.md). But I don't want to add such dependency. 

I also dislike the `pause(1000)`, it will add 1s latency to each test.